### PR TITLE
Project created, sorted

### PR DESCRIPTION
**Pull request by danil-maksimenko**
- [DEMO LINK](https://danil-maksimenko.github.io/layout_catalog/)
- [TEST REPORT LINK](https://danil-maksimenko.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by anstsot — COMMENTED** [review]
I can't find these files to check: 
image

maybe you forgot to push them?

**Review by anstsot — CHANGES_REQUESTED** [review]
Please check the pushed files

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.header {
+  background: $header-background;
+  display: flex;
+  justify-content: space-between;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/header.scss:4]: To ensure that there are always 4 cards in a row, you can use 'grid-template-columns' property with 'repeat' function for the parent container.

File: src/styles/blocks/stars.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: inline-block;
+    width: 16px;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/stars.scss:6]: It is better to use variables for widths and heights, and margins for better maintainability. Consider creating a variable for the star size and using it for width, height, and margin-right.

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
+.card {
+  padding: 16px;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/card.scss:2]: Consider using a variable for the padding, such as $card-padding: 16px;

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $card-color;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/card.scss:5]: Consider using a variable for the fixed card width, such as $card-width: 200px;

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,23 @@
...
+@import "./blocks/content.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/stars.scss";
+
+html {
+  font-family: $main-font;
+}
+
+body,
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/main.scss:13]: Consider adding a line-height to the body element for better readability.

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,23 @@
...
+p,
+h2 {
   margin: 0;
+  padding: 0;
+}
+
+main {
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/main.scss:20]: Please ensure that the 'main' element is used in the HTML file according to the task requirements.

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,443 @@
...
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="content">
+        <div class="content__card card" data-qa="card">
+          <img
+            src="/images/imac.jpeg"
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html:85]: The image src should have a relative path.

File: src/styles/utils/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,12 @@
+$color-accent-blue: rgba(0, 172, 220, 1);
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/utils/_variables.scss:1]: Consider using more descriptive variable names for colors, such as $blue-accent or $primary-blue.

File: src/styles/utils/_variables.scss [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+$color-main-accent: #060b35;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/utils/_variables.scss:2]: Consider using more descriptive variable names for colors, such as $main-text-color or $primary-text-color.

File: src/index.html [hunk 10]
```diff
@@ -5,8 +5,443 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html:16]: why do you need 2 wrappers?

```suggestion
    <header class="header">
```

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
1. should be 4 cards always
image
2. layout looks a little broken if I add some more cards
image

**Pull request closed without merging**